File: java/src/main/java/model/params/OrderBookRawParams.java
package model.params;

import model.ChannelTypes;

public class OrderBookRawParams extends Params {

    private Integer depth;

    public OrderBookRawParams(String symbol, Integer depth) {
        super(ChannelTypes.ORDERBOOKRAW, symbol);
        this.depth = depth;
    }

    public Integer getDepth() {
        return depth;
    }

    public void setDepth(Integer depth) {
        this.depth = depth;
    }

    @Override
    public String toString() {
        return "OrderBookRawParams{" +
                super.toString() +
                ", depth=" + depth +
                "} ";
    }
}

File: c-sharp/ws-client/model/parameters/OrderBookRawParams.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_client.model.parameters
{
    class OrderBookRawParams : Params
    {
        public OrderBookRawParams(string symbol, int? depth) : base(ChannelTypes.ORDERBOOKRAW, symbol)
        {
            this.depth = depth;
        }

        public int? depth { get; set; }
    }
}
